slappd.py

```diff
@@ -127,8 +127,13 @@ def main():
                         checkin['beer']['beer_name'],
                         checkin['brewery']['brewery_name'],
                         checkin['brewery']['brewery_id'],
-                        checkin['brewery']['brewery_slug'],
-                        checkin['rating_score'])
+                        checkin['brewery']['brewery_slug'])
+                
+                # If there's a rating, include it
+                if len(checkin['rating_score']):
+                    text += " ({0}/5)\n".format(checkin['rating_score'])
+                else:
```

Comment:
Thanks for the PR! At some point I was thinking of doing some star rating emoji business, but it seems like more effort than it's worth.